### ISSUE TITLE
Force AWS Region

### ISSUE THREAD

**Issue by metaskills** [customink/crypteia#21]
Seems AWS is having a SSM issue in us-east-1 today. Perhaps we can have a `CRYPTEIA_AWS_REGION` config here https://github.com/customink/crypteia/blob/main/src/ssm.rs#L7 to force another region in case it is needed?

**Comment by jeremiahlukus** [comment]
I think this might have some unintended consequences since regions do not share params.

**Comment by metaskills** [comment]
Yup possible... but the idea here was to give builders a choice. We certainly needed this at one point. Does not come up often tho.
